app/controllers/lookups_controller.rb

```diff
@@ -1,7 +1,12 @@
 class LookupsController < ApplicationController
   def by_base_path
+    # return content_ids for content that is visible on the live site
+    # withdrawn items are still visible
+    states = %w(published unpublished)
```

Comment:
The `Unpublishing` has a [`type`](https://github.com/alphagov/publishing-api/blob/master/app/models/unpublishing.rb#L7-L10) if that helps...